Principal Component Analaysis/PCA.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+class PCA:
+    """
+    PCA: mathematical technique used for dimensionality reduction
+    Attrubutes:
```

Comment:
```suggestion
    Attributes:
```

Reply:
Other than this,it works well and does the calculation just fine as shown in the example.
@geekquad